src/benchcab/internal.py

```diff
@@ -285,3 +318,5 @@ def get_met_forcing_file_names(experiment: str) -> list[str]:
     walltime="01:00:00",
     storage=["gdata/ks32", "gdata/hh5", "gdata/wd9", "gdata/rp23"],
 )
+
+MEORG_PROFILE = {"name": "CABLE", "id": "nFcjg4qqHGPkB9sqE"}
```

Comment:
Why isn't this placed at the end of the file and not together with the experiments ids?